db/migrate/20230217151360_migrate_access_groups_to_access_control_lists.rb

```diff
@@ -0,0 +1,24 @@
+class MigrateAccessGroupsToAccessControlLists < ActiveRecord::Migration[6.1]
+  def up
+    Hmis::Role.all.each do |role|
+      Hmis::AccessGroup.all.each do |access_group|
+        # Users that have this role and are in this group
+        agm_scope = Hmis::AccessGroupMember.where(access_group_id: access_group.id).pluck(:user_id)
+        ushdr_scope = Hmis::UserHmisDataSourceRole.where(role_id: role.id)
+        users = Hmis::User.where(id: agm_scope.pluck(:user_id)).merge(Hmis::User.where(user_id: ushdr_scope.pluck(:user_id)))
```

Comment:
Does this work? `pluck(:user_id)` is being called twice on `agm_scope`

Reply:
Good catch! Fixed